src/workloads/execution/MultiPlanning.yml

```diff
@@ -0,0 +1,148 @@
+- &SmallKey {^Join: {array: ["1", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
+- &BigKey {^Join: {array: ["8", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
+- &SelectiveQuery {^Choose: {
+  from: [
+    {$gte: *BigKey},
+    {$lte: *SmallKey}
+  ]
+}}
+- &NonSelectiveQuery {^Choose: {
+  from: [
+    {$gte: *SmallKey},
+    {$lte: *BigKey}
+  ]
+}}
+
+Actors:
+- Name: InitCollectionAndDropPlanCache
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        drop: *Collection
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *Collection
+        indexes:
+        - key:
+            a: 1
+          name: a_1
+        - key:
+            b: 1
+          name: b_1
+        - key:
+            c: 1
+          name: c_1
+  - &Nop {Nop: true}
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+
+- Name: Insert
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: *docCount
+        BatchSize: 1000
+        Document:
+          a: *Key
+          b: *Key
+          c: *Key
+
+- Name: Lookup
+  Type: CrudActor
+  Database: *Database
+  Threads: 4
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 2000
+    Collection: *Collection
+    Operations:
+    - OperationName: find
+      OperationMetricsName: FindWithMostSelectiveA
+      OperationCommand:
+        Filter: {
+          a: *SelectiveQuery,
+          b: *NonSelectiveQuery,
+          c: *NonSelectiveQuery
+        }
+  - Repeat: 2000
+    Collection: *Collection
+    Operations:
+    - OperationName: find
+      OperationMetricsName: FindWithMostSelectiveB
+      OperationCommand:
+        Filter: {
+          a: *NonSelectiveQuery,
+          b: *SelectiveQuery,
+          c: *NonSelectiveQuery
+        }
+  - Repeat: 2000
+    Collection: *Collection
+    Operations:
+    - OperationName: find
+      OperationMetricsName: FindWithMostSelectiveC
+      OperationCommand:
+        Filter: {
+          a: *NonSelectiveQuery,
+          b: *SelectiveQuery,
+          c: *NonSelectiveQuery
+        }
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone
+      - standalone-all-feature-flags
+      - replica
+      - replica-all-feature-flags
+    branch_name:
+      $gte: v7.0
```

Comment:
Why shouldn't we run this against older branches?

Reply:
Initially I added it because it used internalQueryDisablePlanCache parameter. Now I can remove it.